teporem/HapticHero
# Added Upload to backend

- Currently contains code for uploading audio + generating beatmap from user provided audio
- Only backend for now (testable using Postman)

To-Dos: 
- Confirm that it can work on other devices (might need to install ffmpeg)

Notes: 
- Uploading is kind of slow (few seconds)

How to Test: 
- Start Frontend Server
- Start Backend server
- Go to start screen with upload options
- Upload song, wait for it to upload before clicking play
- Should go to play screen with the selected song's beatmap 
- (ui a bit jank)
- (song audio currently not playing, just beatmap testing)
- Also testable using postman + routes to /upload
